test/src/test/java/hudson/search/SearchTest.java

```diff
@@ -181,41 +194,6 @@ public void testSearch2ProjectsWithSameDisplayName() throws Exception {
         assertFalse(contents.contains(otherDisplayName));
     }
 
-    @Test
-    public void testProjectNamePrecedesDisplayName() throws Exception {
-        final String project1Name = "foo";
-        final String project1DisplayName = "project1DisplayName";
-        final String project2Name = "project2Name";
-        final String project2DisplayName = project1Name;
-        final String project3Name = "project3Name";
-        final String project3DisplayName = "project3DisplayName";
-
-        // create 1 freestyle project with the name foo
-        FreeStyleProject project1 = j.createFreeStyleProject(project1Name);
-        project1.setDisplayName(project1DisplayName);
-
-        // create another with the display name foo
-        FreeStyleProject project2 = j.createFreeStyleProject(project2Name);
-        project2.setDisplayName(project2DisplayName);
-
-        // create a third project and make sure it's not picked up by search
-        FreeStyleProject project3 = j.createFreeStyleProject(project3Name);
-        project3.setDisplayName(project3DisplayName);
-
-        // search for foo
-        Page result = j.search(project1Name);
```

Comment:
Test was reliant on pressing enter and having Jenkins automagically figure out which project you intended - this is no longer possible with the command palette, if there are no results then you'll get a message saying 'No results' instead.